README.md

```diff
@@ -44,6 +46,22 @@ $ npm install tus-node-server
     });
     ```
 
+    using [credentials](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html#constructor-property) (the example below uses [ECSCredentials](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS.html#ECSCredentials-property))
```

Comment:
the first link refers to `AWS.Credentials`, which we don't use, so is that relevant?

The second link states that the `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` or the `AWS_CONTAINER_CREDENTIALS_FULL_URI` environment variable needs to be present for this to work? If so, we should document that.

Lastly, we should explain what `ECSCredentials` does. I'm also wondering if `credentials` can take other classes then  `ECSCredentials`? Do you know?

Reply:
So if your application is gonna run inside ECS container the ECS will inject this env variable on your container and the `AWS.ECSCredentials` class will do the thing to resolve it into Key/Secret.
Yeah `AWS.ECSCredentials` is just one of the subclass of `AWS.Credentials`